tests/__main__.py

```diff
@@ -31,32 +31,27 @@ def _test_ex():
     # Run tests individually to test them and make sure they can be executed individually
     was_successful = True
     try:
-        exec(open("tests/test_base_models.py").read())
+        base_models_main()
     except Exception:
         was_successful = False
 
     try:
-        exec(open("tests/test_sim_result.py").read())
```

Comment:
It looks like there's an uncaught conflict here. We seem to be deleting the sim_result tests

Reply:
I dont know where "test_sim_result" came from, but it is not one of the original tests.

Reply:
Yes, I believe it was added in #100, merged an hour ago.